src/main/java/org/folio/service/orders/flows/update/unopen/UnOpenCompositeOrderManager.java

```diff
@@ -78,12 +78,12 @@ public Future<Void> process(CompositePurchaseOrder compPO, CompositePurchaseOrde
     return updateAndGetOrderWithLines(compPO, requestContext)
       .map(aVoid -> encumbranceWorkflowStrategyFactory.getStrategy(OrderWorkflowType.OPEN_TO_PENDING))
       .compose(strategy -> strategy.processEncumbrances(compPO, poFromStorage, requestContext))
+      .compose(ok -> processInventory(compPO.getCompositePoLines(), deleteHoldings, requestContext))
```

Comment:
changed order because making order as pending should go after all processing steps. For example without this change receiptStatus always was pending in processeInventory() method.